templates/actors/group/member.hbs

```diff
@@ -0,0 +1,116 @@
+<li class="member" data-uuid="{{ uuid }}">
+    <div class="vitals card" {{#if underlay}}style="--underlay: {{ underlay }}"{{/if}}>
+        <div class="portrait">
+            {{#if portrait.isVideo}}
+            <video src="{{ portrait.src }}" autoplay muted playsinline disablepictureinpicture loop></video>
+            {{else}}
+            <img src="{{ portrait.src }}" alt="{{ name }}">
+            {{/if}}
+        </div>
+        <div class="bars">
+            <strong class="name">
+                <a class="rollable" data-action="showDocument" data-uuid="{{ uuid }}">{{ name }}</a>
+            </strong>
+            {{#with system.attributes.hp}}
+            <div class="hp-bar bar">
+                <label>HP</label>
+                <div class="meter hit-points progress" role="meter" aria-valuemin="0"
+                     aria-valuenow="{{ value }}" aria-valuemax="{{ max }}"
+                     style="--bar-percentage: {{ pct }}%"></div>
+                <div class="values">
+                    <span class="value">{{ value }}</span>
+                    <span class="separator">&sol;</span>
+                    <span class="max">{{ max }}</span>
+                </div>
+            </div>
+            {{/with}}
+            {{#with system.attributes.hd}}
+            <div class="hd-bar bar">
+                <label>HD</label>
```

Comment:
Strangely we don't have a hit dice abbreviation localization yet, so you'll need to add this one:

```suggestion
                <label>{{ localize "DND5E.HITDICE.Abbreviation" }}</label>
```